cypress/integration/my_todo_page.spec.ts

```diff
@@ -0,0 +1,97 @@
+import { BookedEvent } from '../../src/types';
+import { ZetkinEventResponse } from '../../src/types/zetkin';
+
+describe('/users/my/todo', () => {
+    beforeEach(() => {
+        cy.request('delete', 'http://localhost:8001/_mocks');
+    });
+
+    let dummyEventResponses : {data: ZetkinEventResponse[]};
+    let dummyBookedEvents : {data: BookedEvent[]};
+
+    before(() => {
+        cy.fixture('dummyEventResponses.json')
+            .then((data : {data: ZetkinEventResponse[]}) => {
+                dummyEventResponses = data;
+            });
+        cy.fixture('dummyBookedEvents.json')
+            .then((data : {data: BookedEvent[]}) => {
+                dummyBookedEvents = data;
+            });
+    });
+
+    after(() => {
+        cy.request('delete', 'http://localhost:8001/_mocks');
+    });
+
+    it('contains to-do events', () => {
```

Comment:
The term _to-do events_ is not something that appears anywhere else in Zetkin. It might make sense to use it in the context of this new feature/page because we call the page "To Do", but if that's a term we want to use, it should be used for _all events that are on the page_, because they are all events that the user should attend, i.e. they're all things _to do_.

It seems here like you are making a distinction between events to which the user responded ("to-do events") and events to which the user has been booked by an organizer ("booked events"). When making that distinction, please use the established terminology of "respond" and "booked" rather than calling just one of them "to-do".